### PR TITLE
specify which path we're matching

### PR DESCRIPTION
**Pull request by jessitron**
This wasn't clear to me, when I run `depcruise --validate .dependency-cruiser.json src`, exactly what I was trying to match in that regex. (relative to here? relative to src? relative to the file, as passed to `import` ?)

Thank you for making this tool and publishing and documenting it! This was really useful yesterday in identifying a circular dependency in TypeScript, which mocha totally barfs on.

> Plan to do something drastic?     
> Leave an [issue](https://github.com/sverweij/dependency-cruiser/issues/new) with a
> summary of the changes you propose + some context on why you'd want to
> do that.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] The code I add will be subject to [The MIT license](../LICENSE), and I'm OK with that.
- [x] The code I've added is my own original work.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](./CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/rules-reference.md [hunk 1]
```diff
@@ -137,6 +137,8 @@ out dependency-cruiser will assume it to be `warn`.
 A regular expression an end of a dependency should match to be catched by this
 rule.
 
+In `from`, this is the path from project root to the file containing a dependency. In `to`, this is the path from project root to the file the dependency resolves to.
```
Comment (ajafff) [inline doc/rules-reference.md]: I wouldn't call this "project root". IMO "current working directory" matches better.

Something like: All paths are relative to the current working directory.

**Comment by jessitron** [comment]
whatever is most accurate! the PR is really a suggestion for better docs
(this was the one thing that frustrated me using it yesterday, everything
else made sense and worked well).

**Comment by sverweij** [comment]
@jessitron thanks for the PR :+1: ! It improves the doc where it was lacking. Sometimes it confuses me as well - but I never had the realization to document it.

File: doc/rules-reference.md (hunk 1 above)
Comment (sverweij) [reply doc/rules-reference.md]: The formulations supplement each other:
- technically dependency-cruiser matches from the current working directory
- typically you'll be working from the root of your project folder. Mentioning that will probably be helpful for first time users.

**Comment by sverweij** [comment]
@jessitron thanks again for the PR - sorry it took so long to merge!

**Pull request merged**